client/src/api/authApi.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+import authService from '../services/authService'; // Import the auth service
+
+// Create Axios instance
+const authApi = axios.create({
+  baseURL: 'http://127.0.0.1:5001/api/v1',
```

Comment:
move this part "http://127.0.0.1:5001" to a global variable or config file if supported may be "process.env.REACT_APP_BASE_URL"
and use it here

if the base including "api/v1" then save all the string